modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -44,6 +44,7 @@ import statusCode from './status_code';
 import addParams from './util/add_params';
 import getCountries from './util/address/get_countries.es';
 import getRegions from './util/address/get_regions.es';
+import Cookie from './util/cookie';
```

Comment:
As I mentioned in another comment, I think it would be cleaner to do `import {getCookie, setCookie, removeCookie} from '.util/cookie';`

And then you just add them to the global namespace:
```js
Liferay.Util.getCookie = getCookie;
Liferay.Util.setCookie = getCookie;
Liferay.Util.removeCookie = removeCookie;
```

This allows you the flexibility even if you must have a Liferay.Util.Cookie scheme, because here you can do the following
```js
Liferay.Util.Cookie.get = getCookie;
Liferay.Util.Cookie.set = getCookie;
Liferay.Util.Cookie.remove = removeCookie;
```

Reply:
I have to disagree there 😅 I think this is cleaner and gets the point across:
```js
import Cookie from './util/cookie';

Liferay.Util.Cookie = Cookie;
```

I see where you're coming from with the flexibility aspect, but I don't think it's useful in this case. This file is a jumble of imports and exports, it's almost 350 lines long already! Keeping it simple would help maintain it.

Reply:
Good decision then! 👍